readme.md

```diff
@@ -3,11 +3,11 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# DevExpress Blazor Components - How to Update CSS Styles to v22.2
+# DevExpress Blazor Components - How to Update CSS Styles
 
 This document summarizes the most common scenarios in which you should use our private CSS selectors to apply a style to an element.
  
-You can also press `Ctrl+F` and search for a private CSS selector that you used in a previous version. This helps you find a selector used in v22.1 or prior, and copy the new equivalent of this selector.
+You can also press `Ctrl+F` and search for a private CSS selector that you used in a previous version. This helps you find a selector used in v22.2 or prior, and copy the new equivalent of this selector.
 
 If you did not manage to find your scenario in this document, you can inspect a component render and create a new CSS selector as described in the following articles: 
```

Comment:
I don't think someone would scroll 2k+ lines to check everything, so I'd prefer to leave these lines here.

```suggestion
If you did not manage to find your scenario in this topic, you can inspect a component render and create a new CSS selector as described in the following articles: 
```